src/marketplace/api.js

```diff
@@ -0,0 +1,58 @@
+import { axiosInstance } from '../api/entities';
+import { MARKETPLACE_URL } from './constants';
+
+const getAccessToken = () =>
+  axiosInstance.post(`${MARKETPLACE_URL}/api-security/om-auth/cloud/token`, {
+    grant_type: 'urn:ibm:params:oauth:grant-type:apikey',
+    apikey: localStorage.getItem('marketplace-key'),
+  });
+
+const query = `
+{
+  productByIds (ids: [
+    "ca54ff2ba24df884a96995f1eefd081b",
+    "ccb29628eabe15c81121ecf231a0a8cd",
+    "9e3b238934c288394680f3a53d1544e0",
+    "bd44b472123b544a630a939c650e968e"
+	]) {
+    products {
+      id
+      productPageName
+      publishedContent {
+        title
+        localizedContent {
+          descriptions {
+            short
+          }
+        }
+        logo {
+          url
+        }
+      }
+      productRating {
+        averageRating
+      }
+      primaryCategory {
+        content { name }
+      }
+      company {
+        displayName
+      }
+    }
+  }
+}`;
+
+let access_token;
```

Comment:
For how long is the token valid? This may cause issues once the token expires.

Reply:
I wasn't actually able to expired it out, it is valid for a long time. However, as it is just a POC and the external api will be replaced with an internal service soon, I wouldn't care too much.